tests/Config/fixtures/complex-config.json

```diff
@@ -0,0 +1,62 @@
+{
+    "storage": {
+        "input": {
+            "tables": [
+                {
+                    "source": "out.c-extrapolation.extrapolation_ini",
+                    "destination": "extrapolation_ini.csv",
+                    "where_column": "",
+                    "where_values": [],
+                    "where_operator": "eq",
+                    "columns": []
+                },
+                {
+                    "source": "in.c-structure.structure",
+                    "destination": "structure.csv",
+                    "where_column": "",
+                    "where_values": [],
+                    "where_operator": "eq",
+                    "columns": []
+                },
+                {
+                    "source": "out.c-marketing.marketing_source_bridge",
+                    "destination": "sources_bridge.csv",
+                    "where_column": "",
+                    "where_values": [],
+                    "where_operator": "eq",
+                    "columns": []
+                }
+            ]
+        },
+        "output": {
+            "tables": [
+                {
+                    "source": "extrapolation_out.csv",
+                    "destination": "in.c-extrapolation.out5"
+                }
+            ]
+        }
+    },
+    "parameters": {
+        "metric_type": "session_granularity",
+        "web_filter": [
+            "czc.cz",
+            "bigbrands.cz",
+            "bigbrands.sk",
+            "bigbrands.pl",
+            "prodeti.cz",
+            "bambino.sk",
+            "mall.cz"
+        ],
+        "web_batch": {
+            "numerator": [
+                6
+            ],
+            "denominator": 7
+        }
+    },
+    "runtime": {
+        "repository": "https://github.com/viktorsohajek/extrapolation",
```

Comment:
dal bych tam fejkovy url

Reply:
Vůbec se to nepoužívá a přehlédl jsem to. Mažu úplně.